freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/keytab/doc/KeytabOperationsDescription.java

```diff
@@ -0,0 +1,9 @@
+package com.sequenceiq.freeipa.api.v1.keytab.doc;
+
+public class KeytabOperationsDescription {
+    public static final String DESCRIBE_SERVICE_CREATE = "Get's the keytab for the provided service on a specific host";
```

Comment:
I think "on a specific host" is redundant with the definition of a service principal. We could probably remove it and be clearer, especially later when this is used for MLX use cases since "pseudo-hosts" will be used that map to clusters not actual hosts (see the MLX section of https://docs.google.com/document/d/1ZSVBYBioX2gliO1UM66rPl4Y83YIB_tZrll3AggqJGs/edit).

Reply:
Service principal is an entity which represents a service running on a specific host. Why do you think it is redundant.

When we get to MLX we can change it accordingly, what do you say?

Reply:
Ok we can update the comment later.